Sources/ArcGISToolkit/Components/Augmented Reality/CalibrationView.swift

```diff
@@ -0,0 +1,193 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ARKit
+import SwiftUI
+import ArcGIS
+
+extension WorldScaleGeoTrackingSceneView {
+    /// A view that allows the user to calibrate the heading of the scene view camera controller.
+    struct CalibrationView: View {
+        /// The view model for the calibration view.
+        @ObservedObject var viewModel: ViewModel
+        /// The scene camera controller heading.
+        @State private var heading: Double = 0
+        /// The scene camera controller elevation.
+        @State private var elevation: Double = 0
+        
+        var body: some View {
+            VStack {
+                HStack(alignment: .firstTextBaseline) {
+                    Text("Calibration")
+                        .font(.title)
+                        .lineLimit(1)
+                    Spacer()
+                    dismissButton
+                        .layoutPriority(1)
+                }
+                .padding(.bottom)
+                headingSlider
+                Divider()
+                elevationSlider
+            }
+            .padding()
+            .background(.regularMaterial)
+            .clipShape(RoundedRectangle(cornerRadius: 15))
+            .frame(maxWidth: 350)
+            .padding()
+            .task {
+                heading = viewModel.cameraController.originCamera.heading
+                elevation = viewModel.cameraController.originCamera.location.z ?? 0
+            }
+        }
+        
+        @ViewBuilder
+        var headingSlider: some View {
+            VStack {
+                HStack {
+                    Stepper() {
+                        HStack {
+                            Text("Heading")
+                                .font(.body.smallCaps())
+                                .foregroundStyle(.secondary)
+                            Spacer()
+                            Text((viewModel.calibrationHeading ?? viewModel.cameraController.originCamera.heading), format: .number.precision(.fractionLength(0)))
```

Comment:
Add in degree symbol

```suggestion
                            Text((viewModel.calibrationHeading ?? viewModel.cameraController.originCamera.heading), format: .number.precision(.fractionLength(0)))
                            + Text("°")
```

Reply:
Consider using a measurement format style, which is best practice for formatting measurements:

```swift
+ Text(
    Measurement<UnitAngle>(value: heading, unit: .degrees),
    format: .measurement(width: .narrow, numberFormatStyle: .number.precision(.fractionLength(0)))
)
```

If you want to, you could even store the heading as a measurement.